app/utils/staticData/desafios.js

```diff
@@ -197,7 +207,7 @@ export const desafios = [
     }
   },
   {
-    id: 22,
+    id: "viejo22",
```

Comment:
Nono, perdón, este debe seguir siendo el 22.
```suggestion
    id: "22",
```